# Experiment 80928c94dbe7a67bc3ee7e94b6679468051685c9e9216b2b012320d6cc1d7207
# Base b18bcbfa99d6ba29452bbaf6114c4b6a9c380cbb
diff --git a/src/pnl_dlg.cpp b/src/pnl_dlg.cpp
index 60130ed36..e4604c7b8 100644
--- a/src/pnl_dlg.cpp
+++ b/src/pnl_dlg.cpp
@@ -1,5 +1,6 @@
 
 #include "../include/pnl_dlg.h"
+#include "../include/dlg_msg.h"
 
 TDialogPanel::TDialogPanel(char* name)
     : TEasy_Panel(name)
